content/package.json

```diff
@@ -106,6 +106,16 @@
   "pnpm": {
     "patchedDependencies": {
       "astro-tweet": "patches/astro-tweet.patch"
+    },
+    "onlyBuiltDependencies": [
+      "@mixedbread/cli",
+      "@parcel/watcher",
+      "@vercel/speed-insights",
+      "esbuild",
+      "msgpackr-extract",
+      "protobufjs",
+      "sharp"
+    ]
     }
```

Comment:
I believe this is causing an issue - can probably fix it by removing the extraneous `}`: 

<img width="2540" height="628" alt="Image" src="https://github.com/user-attachments/assets/5739f16f-b786-4a66-9b14-6fb04ab3dd78" />